gluten-core/src/main/java/io/glutenproject/substrait/rel/LocalFilesNode.java

```diff
@@ -115,6 +104,15 @@ public ReadRel.LocalFiles toProtobuf() {
       if (index != null) {
         fileBuilder.setPartitionIndex(index);
       }
+      Map<String, String> pc = partitionColumns.get(i);
```

Comment:
Can we use a reader-friendly name here also? Please also consider other code part where pc/pcs are used. Thanks!

BTW, please do a final code rebase. Will merge the patch soon if no more comment from others.

Reply:
All done, thanks!